app/screens/search/channel_display_name/index.js

```diff
@@ -16,7 +16,7 @@ function makeMapStateToProps() {
         const channel = getChannel(state, {id: post.channel_id});
 
         return {
-            displayName: channel.display_name,
+            displayName: channel ? channel.display_name : '',
```

Comment:
Might not related to this ticket but this is fix for JS error which occurred when accessing flagged posts